### PR TITLE
doc: README

### PR DESCRIPTION
**Pull request by yesnault**

### CODE REVIEW

**Review by richardlt — APPROVED** [review]

**Review by sguiheux — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-cds#19802.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ CDS is an Enterprise-Grade Continuous Delivery & DevOps Automation Platform writ
 
 
 ## Intuitive UI
-CDS provides an intuitive UI that allows you to build complex workflows, run them and dig into the logs when needed.
+CDS provides an intuitive UI that allows you to build complex workflows, run them, and dig into the logs when needed.
```
Comment (fsamin) [inline README.md:16]:
```suggestion
CDS provides an intuitive UI that allows you to build complex workflows, run them and dig into the logs when needed.
```

File: README.md [hunk 2]
```diff
@@ -103,12 +103,12 @@ You can configure everything with the web UI. Even if you have complex use cases
 
 #### Configuration on Git Repository
 
-Pipeline as code is a well-known concept of CI / CD tools. CDS goes a step further and offers workflow as code. This is done by git-pushing using yaml configuration files of your workflow (+ pipeline, + applications, + environment). This is particularly useful as you can test your new workflow on a dev branch, before merging the changes on the master branch.
+Pipeline as code is a well-known concept of CI / CD tools. CDS goes a step further and offers workflow as code. This is done by git-pushing using .yaml configuration files of your workflow (+ pipeline, + applications, + environment). This is particularly useful as you can test your new workflow on a dev branch, before merging the changes on the master branch.
```
Comment (fsamin) [inline README.md:106]: not really because the files are yaml files but must be .yml

File: README.md [hunk 3]
```diff
@@ -103,12 +103,12 @@ You can configure everything with the web UI. Even if you have complex use cases
...
 
 
 #### Configuration as code on UI
 
-You can modify your workflow with the UI, you can also modify the configuration by editing the yaml directly in the UI if you wish. This is an excellent way to learn how to use the workflow-as-code feature.
+You can modify your workflow with the UI, you can also modify the configuration by editing the .yaml file directly in the UI if you wish. This is an excellent way to learn how to use the workflow-as-code feature.
```
Comment (fsamin) [inline README.md:111]: same

File: README.md [hunk 4]
```diff
@@ -156,18 +156,18 @@ This event flow can then feed other services (reporting, notifications, etc., ).
...
 Ability to manage multiple environments (e.g. dev/prod/staging) in a secure way with segregated access rights.
-In practice, an environment is a just set of variables that you can use within your workflows.
+In practice, an environment is a set of variables that you can use within your workflows.
 
 With CDS, You can use a deployment pipeline on your preproduction environment and use that same deployment pipeline on your production environment. The ability to deploy to production can be limited to a pre-established group of users.
 
 #### Enterprise-grade permissions / Support of ACLs delegation
 
-Users are free to create groups and manage users in their groups. A group can have the rights to read, write, execute on their projects and their workflows. You can also restrict the execution of some pipelines to some groups if you wish.
+Users are free to create groups and manage users in their groups. A group can have the right to read, write, and execute their projects and their workflows. You can also restrict the execution of some pipelines to some groups if you wish.
```
Comment (fsamin) [inline README.md:170]:
```suggestion
Users are free to create groups and manage users in their groups. A group can have the right to read, write and execute their projects and their workflows. You can also restrict the execution of some pipelines to some groups if you wish.
```

File: README.md [hunk 5]
```diff
@@ -188,36 +188,36 @@ CDS projects are isolated from one another, but the same group may have access r
 #### Execution Environment Customization
 
 A worker model is a worker execution context. Let's say, you need to run a job that requires GoLang v1.11.5. In CDS, you just need to create a Go worker model, containing a go in version 1.11.5.
-A worker model can be a docker image, an openstack image, a VSphere image.
+A worker model can be a docker image, an OpenStack image, or a VSphere image.
```
Comment (fsamin) [inline README.md:191]:
```suggestion
A worker model can be a docker image, an OpenStack image or a VSphere image.
```

File: README.md [hunk 6]
```diff
@@ -188,36 +188,36 @@ CDS projects are isolated from one another, but the same group may have access r
...
+After reading the previous points, you've understood: self-service is everywhere. All users can create their project/workflow/ worker models/workflow templates/actions... And run Jobs in a totally isolated environment. CDS projects are builders, on which you can add integrations. All this allows you to have only one instance of CDS for your entire company.
 
-#### Command Line Interface (cdsctl): 100% features supported
+#### Command Line Interface (cdsctl): 100% of features supported
 
-All you can do with the UI is available via the Command-Line Interface (CLI), named "cdsctl". cdsctl is available on all the OS: darwin, freebsd, linux, openbsd ... cdsctl will allow you to create, launch, export, import your workflows, monitor your CDS, navigate through your projects, workflows.
+All you can do with the UI is available via the Command-Line Interface (CLI), named "cdsctl". cdsctl is available on all the OS: Darwin, FreeBSD, Linux, OpenBSD... cdsctl will allow you to create, launch, export, import your workflows, monitor your CDS, and navigate through your projects, and workflows.
```
Comment (fsamin) [inline README.md:205]:
```suggestion
All you can do with the UI is available via the Command-Line Interface (CLI), named "cdsctl". cdsctl is available on all the OS: Darwin, FreeBSD, Linux, OpenBSD... cdsctl will allow you to create, launch, export, import your workflows, monitor your CDS and navigate through your projects and workflows.
```

**Review by fsamin — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19802.0 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

File: README.md (hunk 2 above)
Comment (yesnault) [inline README.md:106]:
```suggestion
Pipeline as code is a well-known concept of CI / CD tools. CDS goes a step further and offers workflow as code. This is done by git-pushing using yaml configuration files of your workflow (+ pipeline, + applications, + environment). This is particularly useful as you can test your new workflow on a dev branch, before merging the changes on the master branch.
```

File: README.md (hunk 3 above)
Comment (yesnault) [inline README.md:111]:
```suggestion
You can modify your workflow with the UI, you can also modify the configuration by editing the yaml file directly in the UI if you wish. This is an excellent way to learn how to use the workflow-as-code feature.
```

**Comment by ovh-cds** [comment]
CDS Report build-cds#19810.0 ✔ 
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19810.1 ✘  
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✘  
  * Standalone Tests v2 ✔

**Comment by ovh-cds** [comment]
CDS Report integration-cds#19810.2 ✔ 
* Integration
  * Docker Compose Tests ✔ 
  * Standalone Tests ✔ 
  * Standalone Tests v1 ✔ 
  * Standalone Tests v2 ✔

**Pull request merged**